pkg/controller/statusmanager/status_manager.go

```diff
@@ -300,9 +300,7 @@ func (status *StatusManager) SetFromPods() {
 
 		// Finally check whether Pods belonging to this Deployments are being started or they
 		// are being scheduled.
-		if dep.Status.UnavailableReplicas > 0 {
-			progressing = append(progressing, fmt.Sprintf("Deployment %q is not available (awaiting %d nodes)", depName.String(), dep.Status.UnavailableReplicas))
-		} else if dep.Status.AvailableReplicas == 0 {
+		if dep.Status.AvailableReplicas == 0 {
```

Comment:
This depends on the nature of the app you are installing, some have leader elections other don't, maybe we have to make that configurable or check it.